pkg/registry/clusteroperator/machine/storage.go

```diff
@@ -136,11 +136,11 @@ func NewStorage(opts etcd.Options) (nodes, nodesStatus rest.Storage) {
 		// Used to match objects based on labels/fields for list.
 		PredicateFunc: Match,
 		// DefaultQualifiedResource should always be plural
-		DefaultQualifiedResource: clusteroperator.Resource("clusterservicenodes"),
+		DefaultQualifiedResource: clusteroperator.Resource("clusterservicemachines"),
```

Comment:
This is a mistake in the update from the code that was brought over from service catalog. This should be "machines" not "clusterservicemachines".